src/frontend/Preview.tsx

```diff
@@ -29,8 +29,18 @@ const PreviewApp = observer(() => {
     [fileStore.fileList.length, uiStore],
   );
 
+  // disable fade-in on initalization (when file list changes)
```

Comment:
Shouldn't this be part of the zoom component?

Reply:
mmhh well, it's behavior specific to the preview window, so I wasnt sure where to put it, otherwise we'd need to check `if (IS_PREVIEW_WINDOW)` in there somewhere, which isn't pretty either